mkosi/run.py

```diff
@@ -514,36 +519,49 @@ def sandbox_cmd(
 
         if d and not any(Path(d).is_relative_to(dir) for dir in (*dirs, "/usr", "/nix", "/tmp")):
             cmdline += ["--bind", d, d]
+    else:
+        cmdline += ["--dir", "/var/tmp", "--unshare-ipc"]
 
-    for d in ("bin", "sbin", "lib", "lib32", "lib64"):
-        if (p := tools / d).is_symlink():
-            cmdline += ["--symlink", p.readlink(), Path("/") / p.relative_to(tools)]
-        elif p.is_dir():
-            cmdline += ["--ro-bind", p, Path("/") / p.relative_to(tools)]
+        if devices:
+            cmdline += ["--bind", "/sys", "/sys", "--bind", "/dev", "/dev"]
```

Comment:
Previously in the `relaxed=False and devices=True` case, this would have bind mounted `/run` as well, but is not now or am I missing something?

Reply:
Yeah that's a bug fix, there's no need to mount /run when we need device access.

Reply:
Excellent! Could you add that to the commit message in case anybody is following along?